### ISSUE TITLE
TStructureItem.extensions

### ISSUE THREAD

**Issue by Shimon42** [Amphore-Dev/Architect#9]
String | TExtensions

Extension.index
Extension.default

**Comment by Shimon42** [comment]
#11

**Comment by Shimon42** [comment]
```
"structure": {
    "components": {
        "extensions": {
            "default": "DEFF",
            "index": "INDEX"
        },
	"generateSubIndex": true,
	"generateSubdirs": true,
	"subdirs": {
            "atoms": "atom",
            "molecules": "molecule",
            "organisms": "organism",
            "pages": "page",
            "templates": "template"
        },
        "type": "component"
    }
}
```

Bug: Extensions defined in the "components" are not used for subdirs
- [ ] Handle parent config

**Comment by Shimon42** [comment]
fixed in #12
